frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx

```diff
@@ -107,11 +107,8 @@ export function launchLogs(cluster: Cluster, configMaps: ConfigMap[]) {
     const response = getHivePod(cluster.namespace, cluster.name, cluster.status)
     response.then((job) => {
       const podName = job?.metadata.name || ''
-      const containerName = podName.includes('uninstall') ? 'deprovision' : 'hive'
-      podName &&
-        window.open(
-          `${openShiftConsoleUrl}/k8s/ns/${cluster.namespace}/pods/${podName}/logs?container=${containerName}`
-        )
+      const containerName = podName.includes('uninstall') ? '?container=deprovision' : ''
```

Comment:
Can you check if we can remove the container name for deprovision as well?

Reply:
It looks like the default for the uninstall pod also works.